package.json

```diff
@@ -98,8 +99,6 @@
   },
   "lint-staged": {
     "*.ts": [
-      "eslint --fix",
```

Comment:
Question: why are we removing these line?

Reply:
not to worry, quick dev things. will add those back..